src/models/enums.ts

```diff
@@ -0,0 +1,83 @@
+export enum Grade{
+  UNKNOWN,
+  HIGHSCHOOL_JUNIOR,
+  HIGHSCHOOL_SENIOR,
+  COLLEGE_FRESHMAN,
+  COLLEGE_SOPHOMORE,
+  COLLEGE_JUNIOR,
+  COLLEGE_SENIOR
+}
+export enum Gender{
```

Comment:
I'm also surprised it didn't complain about the spacing between `enum`s. Does `yarn lint --fix` fix this?